src/js/main.js

```diff
@@ -14,22 +14,25 @@ export function init(el, context, config) {
             var network = shareEl.getAttribute('data-network');
             shareEl.addEventListener('click', () => shareFn(network));
         });
-
         const youTubeId = resp.sheets[config.sheetName][0].youTubeId;
+        const chapters = resp.sheets[config.sheetChapter];
 
         getYouTubeVideoDuration(youTubeId, function(duration) {
             builder.querySelector('.docs__poster--play-button').setAttribute('data-duration', duration);
         });
 
-        pimpYouTubePlayer(youTubeId, builder.querySelector('#playerWrapper'), '100%', '100%');
-
         var hiddenDesc = builder.querySelector('.docs--standfirst-hidden');
         var showMoreBtn = builder.querySelector('.docs--standfirst-read-more');
 
         var hiddenAbout = builder.querySelector('.docs--about-wrapper');
         var showAboutBtn = builder.querySelector('.docs--sponsor-aboutfilms');
         var hideAboutBtn = builder.querySelector('.docs--about-wrapper');
 
+        var chapterButtons = builder.querySelector('.docs--chapters');
+        chapters.forEach( function(chapter){
+          chapterButtons.innerHTML += '<li class="" data-sheet-attribute="'+ chapter.chapterTimestamp +'">' + chapter.chapterTitle + '</li>';
```

Comment:
for a subsequent PR, it may be better to use `appendChild` rather than concatenating to `innerHTML`